### DISCUSSION TITLE
multi timeframe

### DISCUSSION THREAD

**Discussion by starbuzz** [highfestiva/finplot#97] [Q&A]
Hello,

Do you think there is a way to display multiple timeframes on other axes ?

**Comment by highfestiva** [comment]
Yes, absolutely. Using https://github.com/highfestiva/finplot#disable-zoompan-sync-between-axes you can have the two axes totally separated.

**Reply by starbuzz** [reply]
I tried to have candlestick_ochl on ax1 with time frame 5min, sync with an other candlestick_ochl on ax2 with a greater time frame like 1 hour. It does not seem to work, even desync is not really working. 
Is a figure supposed to have maximum one candlestick_ochl ?

Great job by the way, the most powerfull lib I tested.

**Reply by highfestiva** [reply]
Possibly you are on the bleeding edge too, see #84. If that does not solve it, please post a small and complete example with a description of the error and I'll hunt for the bug.
